Sources/Basic/PackageDir.swift

```diff
@@ -0,0 +1,28 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import Utility
+
+public func isSafeToRemove(_ item: String) -> Bool {
+    // Only look at repositories.
+    guard Path.join(item, ".git").exists else { return true }
```

Comment:
I think this method should assume it is called on a repository, most clients should know that it just happens the check of what is in `Packages` doesn't.